app/Http/Controllers/DemandController.php

```diff
@@ -112,7 +112,11 @@ public function serve(Request $request, $id): Response
         if ('OPTIONS' === $request->getRealMethod()) {
             $response = new Response('', Response::HTTP_NO_CONTENT);
         } else {
-            $response = new GzippedStreamedResponse();
+            if (str_starts_with($banner->creative_mime, 'text')) {
```

Comment:
PHP 8 only

Reply:
Hmm działa. Może przez to: Symfony\Polyfill\Php80